app/models/validations/sales/soft_validations.rb

```diff
@@ -43,6 +43,18 @@ def extra_borrowing_expected_but_not_reported?
     extrabor != 1 && mortgage + deposit > value - value * discount / 100
   end
 
+  def hodate_3_years_or_more_exdate?
+    return unless hodate && exdate
+
+    ((exdate.to_date - hodate.to_date).to_i / 365) >= 3
```

Comment:
I bet ruby has a fancy way of doing this

Reply:
not related to this ticket, this change was removed after rebase
agreed though, have railsified